projects/canopy/src/lib/modal/modal.component.scss

```diff
@@ -0,0 +1,54 @@
+@import '../../styles/mixins';
+
+.lg-modal-overlay {
+  background-color: var(--overlay-color);
+  height: 100%;
+  left: 0;
+  overflow: auto;
+  position: fixed;
+  text-align: center;
+  top: 0;
+  width: 100%;
+  z-index: 1000;
+}
+
+.lg-modal-overlay:before,
+.lg-modal {
+  display: inline-block;
+  text-align: left;
+  vertical-align: middle;
+}
+
+.lg-modal {
+  animation-duration: var(--animation-duration);
+  animation-name: opacity;
+  left: 50%;
+  max-height: 100%;
+  overflow-y: auto;
+  position: fixed;
+  top: 50%;
+  transform: translate(-50%, -50%);
+  width: 90%;
+  z-index: 1001;
```

Comment:
Perhaps we should store the z-index as a var in the main variables file? I'm thinking of when we have other items on the z-index, such as popovers/tooltips etc, we could ensure the modal is always the highest number. Could be something like `--zindex-modal`, then later the could be a `--index-tooltip`, which we would ensure is lower.

Reply:
Good call, we should probably have some set z-indexes for certain tasks.

LIghtning do this.. https://www.lightningdesignsystem.com/design-tokens/#category-z-index

Might be some insight in here too..
https://www.smashingmagazine.com/2021/02/css-z-index-large-projects/

Reply:
Yes, nice one 👍

Reply:
I've added them to the main variables file. As we add more we could split them in a separate file.